src/Materials/Textures/Procedurals/proceduralTexture.ts

```diff
@@ -152,6 +163,11 @@ export class ProceduralTexture extends Texture {
         return this._effect;
     }
 
+    /** @hidden */
+    public _setEffect(effect: Effect) {

```

Comment:
Is this needed? You don't use it in `NodeMaterial.createProceduralTexture`.

Reply:
lol I directly use _effect. Let me remove that